employees.rb

```diff
@@ -0,0 +1,103 @@
+class Employee
+  attr_accessor :full_name
+  attr_accessor :id
+
+  def initialize(full_name, id)
+    @full_name = full_name
+    @id = id
+  end
+
+  def surname
+    @full_name.split(' ', 2).last
+  end
+
+  def first_name
+    @full_name.split(' ', 2)[0]
```

Comment:
Just to be consistent
```suggestion
    @full_name.split(' ', 2).first
```